tips/templates/tips/board.html

```diff
@@ -3,57 +3,59 @@
 <html lang="en">
```

Comment:
I left comments about this code in @Yoavsc129 
please make sure you are pulling the code after the fix from him
https://github.com/redhat-beyond/Xpense/pull/71#discussion_r852185431